README.md

```diff
@@ -154,7 +160,9 @@ it must be called in `content_by_lua*`.
 
 publish
 ----
-`syntax: ok, err = ev:publish(target, source, event, data)`
+**syntax:** *ok, err = ev:publish(target, source, event, data)*
+
+**context:** *all phases*
```

Comment:
Is this accurate? For example, can this support `init` phases?

Reply:
Thank you, I fix it.